.env

```diff
@@ -81,6 +81,9 @@ _APP_USAGE_DATABASE_INTERVAL=15
 _APP_USAGE_STATS=enabled
 _APP_LOGGING_PROVIDER=
 _APP_LOGGING_CONFIG=
+_APP_REGION_DEFAULT=default
```

Comment:
@lohanidamodar @christyjacob4 we might want to use a key value syntax here for having proper labels we can use in the UI.

New York=nyc001,Frankfurt=fra001

WDYT?

Reply:
@lohanidamodar we need to check if there is any way to support spaces in var values.